lib/Map/LeafletDataSourceDisplay.js

```diff
@@ -196,6 +197,39 @@ LeafletDataSourceDisplay.prototype.update = function(time) {
     return result;
 };
 
+/**
+ * Computes the rectangular bounds which encloses the visualization
+ * produced for the given entities.
+ *
+ * @param {EntityCollection} entities The collection of entities whose bounds to compute.
+ * @returns {LatLngBounds} The computed bounds.
+ */
+LeafletDataSourceDisplay.prototype.getLatLngBounds = function(entities) {
+    var dataSources = this._dataSourceCollection;
+    var latLngBounds = [];
+
+    for (var i = 0, len = dataSources.length; i < len; i++) {
+        var dataSource = dataSources.get(i);
+
+        if (entities.id === dataSource._entityCollection.id) {
+            var visualizers = dataSource._visualizersByDisplayID[this._displayID];
+            
+            for (var j = 0, vLength = visualizers.length; j < vLength; j++) {
+                var visualizer = visualizers[j];
+                var bounds = visualizer.getLatLngBounds();
+    
+                if (defined(bounds)) {
+                    latLngBounds.push(bounds);
```

Comment:
Not a big deal at all, but it would be slightly better (faster, less memory usage) to compute the bounds as you go with extend, rather than pushing all the bounds into an array and using `combineLatLngBounds`.

For bonus points, do this as a one-liner (eh, maybe two) with a `reduce` over the visualizers array.